modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -72,6 +72,19 @@
     '/etc/lightdm/session-cleanup':
       mode    => '0755',
       source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/session-cleanup';
+    '/opt/share/xsession/lightdm-gtk-greeter.po':
+      source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/lightdm-gtk-greeter.po';
+  }
+
+  # overwrite greeter strings with OCF ones
+  package {'gettext':;}
+
+  exec { 'lightdm-greeter-compile-po':
+      command   => 'msgcat /opt/share/xsession/lightdm-gtk-greeter.po | msgfmt -o \
+                      /usr/share/locale/en_US/LC_MESSAGES/lightdm-gtk-greeter.mo -',
+      subscribe => File['/opt/share/xsession/lightdm-gtk-greeter.po'],
+      refreshonly => true,
+      require   => Package['lightdm-gtk-greeter'];
```

Comment:
surprised puppet-lint doesn't complain at the arrow alignment

this should probably require gettext too, right?

`require   => Package['lightdm-gtk-greeter', 'gettext']` should work